finger_initial_offset: [0.0,0.0,0.0,0.0]
finger_opening_position: [1.0,0.35,0.42,0.38]
finger_closing_position: [0.0,0.0,0.0,0.0]
finger_move_offset: [0.05,0.065,0.065, 0.065]
finger_close_load: 140.0
